yogstation/code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -101,4 +101,160 @@
 		var/turf/T = get_turf(src)
 		return T.return_air()
 	else
-		return null
\ No newline at end of file
+		return null
+		
+		
+		
+		
+		
+		
+		
+
```

Comment:
aaAaaAAAA thats a lot of empty lines

Reply:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA